portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -1760,242 +2063,286 @@ definition {
+			WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
 
-		CKEditor.assertSourceContent();
+			CKEditor.assertSourceContent();
+		}
 	}
 
 	@description = "This is a use case for LPS-84290. Verify is possible export Home Folder Permissions."
 	@priority = "3"
 	test ImportSiteHomeFolderPermissions {
-		JSONGroup.addGroup(groupName = "Site Name");
+		task ("Given: User adds a new site") {
+			JSONGroup.addGroup(groupName = "Site Name");
+		}
 
-		Navigator.openURL();
+		task ("When: User removes documents and media permissions from Site member on default site") {
+			Navigator.openURL();
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		Portlet.gotoPortletOptions(portletOption = "Home Folder Permissions");
+			Portlet.gotoPortletOptions(portletOption = "Home Folder Permissions");
 
-		SelectFrameTop();
+			SelectFrameTop();
 
-		IFrame.selectModalFrame();
+			IFrame.selectModalFrame();
 
-		Role.removePermissionPG(
-			permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_ADD_SHORTCUT_CHECKBOX",
-			roleTitle = "Site Member");
+			Role.removePermissionPG(
+				permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_ADD_SHORTCUT_CHECKBOX",
+				roleTitle = "Site Member");
 
-		Role.removePermissionPG(
-			permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_ADD_DOCUMENT_CHECKBOX",
-			roleTitle = "Site Member");
+			Role.removePermissionPG(
+				permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_ADD_DOCUMENT_CHECKBOX",
+				roleTitle = "Site Member");
 
-		Role.removePermissionPG(
-			permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_SUBSCRIBE_CHECKBOX",
-			roleTitle = "Site Member");
+			Role.removePermissionPG(
+				permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_SUBSCRIBE_CHECKBOX",
+				roleTitle = "Site Member");
 
-		Role.removePermissionPG(
-			permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_VIEW_CHECKBOX",
-			roleTitle = "Site Member");
+			Role.removePermissionPG(
+				permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_VIEW_CHECKBOX",
+				roleTitle = "Site Member");
 
-		Role.removePermissionPG(
-			permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_ADD_FOLDER",
-			roleTitle = "Site Member");
+			Role.removePermissionPG(
+				permissionDefinitionKey = "CONFIGURATION_PERMISSIONS_ADD_FOLDER",
+				roleTitle = "Site Member");
+		}
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+		task ("And: Exports the LAR file of Documents and Media admin page") {
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		Refresh();
+			Refresh();
 
-		LAR.exportPortlet(
-			exportPermissions = "true",
-			larFileName = "DM.portlet.lar");
+			LAR.exportPortlet(
+				exportPermissions = "true",
+				larFileName = "DM.portlet.lar");
+		}
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "site-name");
+		task ("And: Imports the LAR file to the freshly created site") {
```

Comment:
Need to point out that import the site with the "Import permissions" enabled.

Reply:
This will be verified in the next step, in the "Then" phase